content/millennium/faq/common-issues.md

```diff
@@ -326,7 +326,7 @@ there is potential that the app instance will share patient context and
 show same patient’s data on two separate charts. In order to avoid this,
 please read our [SMART topic on sessionStorage](/smart/#html5-session-storage).
  
-If your application uses the open sourcce fhir-client.js library, please
+If your application uses the open source fhir-client.js library, please
 read the [secion on minimum required version and additional code](/smart/#open-source-fhir-client-libraries)
```

Comment:
Since you're fixing spelling, how about "secion" => "section" :)

Reply:
Good catch: https://github.com/cerner/fhir.cerner.com/pull/218/commits/13c1b9b71e24d54a769898c9f083969fe189b978